source/stylesheets/pages/_home.scss

```diff
@@ -74,13 +77,17 @@ section.intro {
 // About
 
 section#about {
-  background: $accent-color;
-  @include grid-12;
-  transition: height .25s ease;
-  height: 0;
+  transition: all .5s ease;
   overflow: hidden;
-  &.slide-open {
-    height: 100%;
+  max-height: 0px;
+  padding: 0;
+  &.slide {
+    max-height: 100vh;
```

Comment:
We're using the `max-height` property for this animation instead of the default `height` property. By using `max-height` we can rely on some implicit calculations for what the sections height should automatically be whereas the `height` property is more strict and removes our ability to infer

Reply:
oh really great point